### DISCUSSION TITLE
JDT Sync Call Meetings

### DISCUSSION THREAD

**Discussion by rgrunber** [eclipse-jdt/eclipse.jdt.core#2267] [General]
The optional, JDT Sync Call on the first Tuesday of every month at 08:30 EDT (UTC -04:00).

For each call, I'll create a thread and those interested can indicate that they will be attending and have an item to bring up. If no one indicates any items to bring up as part of the agenda, the call can be skipped.

**Comment by rgrunber** [comment]
**2024-08-06**

Please comment inline for any proposals to the agenda.

**Reply by iloveeclipse** [reply]
We should discuss dropping old JLS support, see https://github.com/eclipse-jdt/eclipse.jdt.core/discussions/820

My wish would be to drop everything below 1.7 JLS for 4.33 release.

**Comment by rgrunber** [comment]
**2024-09-03**

Please comment inline for any proposals to the agenda.

**Reply by rgrunber** [reply]
@akurtakov , @iloveeclipse , do we still want to keep this call ? Is https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/wiki/Eclipse-IDE-%E2%80%90-Developers-Community-Call#meeting-minutes an alternative ?

**Reply by iloveeclipse** [reply]
@rgrunber : I'm OK with that

**Reply by HannesWell** [reply]
Your participation would be more than welcome. :)
